app/models/work.rb

```diff
@@ -1,3 +1,16 @@
+# frozen_string_literal: true
+
 class Work < ApplicationRecord
+  has_many :votes
+  acts_as_votable
+
+  def votes
+    votes = Vote.where('votable_id = ?', id)
+    puts 'vote.all.count', Vote.all.count
+    votes
+  end
 
+  def self.upvote(id)
+    puts id.to_s
+  end
```

Comment:
Hm... I'm not sure what this method is... is this part of the `act_as_votable` gem? That being said, would we want it to just `puts id.to_s`? I don't know too many reasons why a `puts` statement would be valuable to the Rails app as a whole